tests/console/yast2_cmdline.pm

```diff
@@ -40,8 +42,25 @@ sub run_yast_cli_test {
 }
 
 sub run {
+    my $cmd;
```

Comment:
Common style so far was to declare the variable as near as possible where used the first time so please move before the `if (is_sle)`